tests/open_alchemy/test_build.py

```diff
@@ -3,6 +3,7 @@
 
 from open_alchemy import build
 from open_alchemy import exceptions
+from open_alchemy.build import PackageFormat
```

Comment:
Is this import needed if `build` is already imported?

Reply:
Nope, I removed it ✅